### ISSUE TITLE
To Do:

### ISSUE THREAD

**Issue by amritbhanu** [ai-se/LDAClassification#6]
## For Research
- [x] Does stability help classification? Tuned and untuned? Multigoal fscore and jaccard score
- [ ] **IMP**: Ldaga paper reproduction (https://dibt.unimol.it/reports/LDA-GA/)
- [x] And topic matching with lda and with jist brute and force 7,10 words
## For LN
- [x] Tune svm and lda, just for fscore (kernel, C, degree, learning rate)
- [ ] Lda+word vector which will have topic word features (Part of my research as well) 
- [x] Mutate Tags by x% to see what error it throws. (Heat Map with tags and topics for different datasets).
## Conclusion:
- Feature engineering, LDA features are much better for text mining rather than tf, tfidf. (Can be a paper)

**Comment by amritbhanu** [comment]
- [ ] tfidf weight + kmeans clustering (Raymond Mooney (Learning Scripts for Text Understanding with Recurrent Neural Networks) seminar title)
- [ ] lda features + kmeans

## LN:
- Std error types
- inter document similarity
- long documents, ambiguous, which makes it confusing
- cosine, with some threshold
- tar3 learner contrast
- http://unbox.org/stuff/tags/tar/3.0/c/

**Comment by amritbhanu** [comment]
PAPERS WHICH CAN BE MADE OUT OF IT:
- [x] LDA + Classification
- [x] LDA features compared with tf , tfidf (Compared with other text mining results) and it performs better with DE
- [ ] LDADE results compared to LDAGA

**Comment by timm** [comment]
"LDA features compared with tf , tfidf (Compared with other text mining results) and it performs better with DE" is probably a small section in external validity

lets see what icse says but i think you have **one** paper showing that stability is useful for supervised and unsupervised learning

i long to want  your stats tests on f2, f1, jacard before and after tuning.

**Comment by amritbhanu** [comment]
- [x] magic parameters tune. New set of examples they want surely classified and not surely classified.
  - waiting for results
